### ISSUE TITLE
Cross-Browser Automated Accessibility Testing in WPT Interop 2023 and Beyond

### ISSUE THREAD

**Issue by cookiecrook** [w3c/tpac2023-breakouts#53]
### Session description

The [Interop 2023 Accessibility Investigation](https://github.com/web-platform-tests/interop-2023-accessibility-testing) has yielded a new implementation-independent accessibility testing infrastructure, and [over 500 new automated accessibility tests](https://wpt.fyi/results/?label=master&label=experimental&aligned&q=label%3Aaccessibility)… And we aren't done yet. 

In 2023, contributors from Apple, Mozilla, Google, Bocoup, Igalia, Adobe, and more kickstarted new work that is now actively running in every major browser engine.

Come hear a summary of [the work so far](https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/3), learn how to contribute new tests, and discuss future proposals (such as [AOM #197](https://github.com/WICG/aom/issues/197)) for improving automated, sustainable accessibility testing beyond what is possible today.

### Session goal

Gather feedback on current automated accessibility testing, and extension proposals for WebDriver and other test interfaces…

### IRC channel (Optional)

#interop-2023-a11y

### Who can attend

Anyone may attend (Default)

### Session duration

60 minutes (Default)

### Other sessions where we should avoid scheduling conflicts (Optional)

#14, #33, #34, #60

### Estimated number of in-person attendees

20-45 people

### Instructions for meeting planners (Optional)

Avoid scheduling conflict with any other BTT, WebDriver, or Interop testing sessions.

### Agenda, minutes, slides, etc. (Optional)

- [Agenda](https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/59)
- Slides: @@
- [Minutes](https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/59)
- [Calendar](https://www.w3.org/events/meetings/7ea3022f-4424-45da-ba98-2953baf7d997/)

**Comment by cookiecrook** [comment]
FYI @jcsteh, @aleventhal, @alice, @cyns, @josepharhar, @gsnedders, @benbeaudry, sorry if I missed anyone

**Comment by cookiecrook** [comment]
Schedulers, you don't need to avoid the other accessibility track sessions except for:

- #34 
- #14 (not tracked under accessibility yet, but should be in addition to web components)

The other sessions to avoid conflicts with are: 

- #33 
- #60

**Comment by cookiecrook** [comment]
Scheduled for Wednesday Sept 13th, 12:15–13:15 CEST.
https://www.w3.org/events/meetings/7ea3022f-4424-45da-ba98-2953baf7d997/

**Comment by cookiecrook** [comment]
Perhaps @ianbjacobs wanted the minutes linked before closing: 
https://www.w3.org/2023/09/13-interop-2023-a11y-minutes.html

**Comment by ianbjacobs** [comment]
I've reopened the issue; we plan to keep all the session issues open. Next year we'll use a new repo. Thanks!
